tst/EnergyPlus/unit/SimAirServingZones.unit.cc

```diff
@@ -0,0 +1,170 @@
+// EnergyPlus::SimAirServingZones Unit Tests
+
+// Google Test Headers
+#include <gtest/gtest.h>
+
+// C++ Headers
+#include <cassert>
+#include <cmath>
+#include <string>
+
+// ObjexxFCL Headers
+#include <ObjexxFCL/Array.functions.hh>
+#include <ObjexxFCL/Fmath.hh>
+#include <ObjexxFCL/gio.hh>
+
+// EnergyPlus Headers
+#include <SimAirServingZones.hh>
+#include <BranchInputManager.hh>
+#include <DataAirLoop.hh>
+#include <DataAirSystems.hh>
+#include <DataContaminantBalance.hh>
+#include <DataConvergParams.hh>
+#include <DataDefineEquip.hh>
+#include <DataEnvironment.hh>
+#include <DataGlobals.hh>
+#include <DataHeatBalance.hh>
+#include <DataHVACControllers.hh>
+#include <DataHVACGlobals.hh>
+#include <DataLoopNode.hh>
+#include <DataPrecisionGlobals.hh>
+#include <DataSizing.hh>
+#include <DataSystemVariables.hh>
+#include <DataZoneEquipment.hh>
+#include <DesiccantDehumidifiers.hh>
+#include <EMSManager.hh>
```

Comment:
HI @mbadams5  You are right, my unit tests don't need to include all these heads, which I copied from "SimAirServingZones.cc". The reason I included all of them here is to save my time and to facilitate other developers who may add more unit tests for "SimAirServingZones.cc". "SimAirServingZones.cc" is pretty large and I think more unit tests will come here.